tools/wpt/revlist.py

```diff
@@ -0,0 +1,167 @@
+                "daily, weekly or 3600, 7200, 10800 ...",
+        "parser": parser_epoch
+    },
+    "max_count": {
+        "name": "max-count",
+        "default": DEFAULT_MAX_COUNT,
+        "help": "Maximum number of revisions to be returned by the command "
+                "(default: %s)." % DEFAULT_MAX_COUNT,
+        "parser": parser_max_count
+    }
+}
+
+
+def get_tagged_revisions(pattern):
+    # type: (bytes) -> List[..., Dict]
+    '''
+    Returns the tagged revisions indexed by the committer date.
+    '''
+    git = get_git_cmd(wpt_root)
+    command = [
+        pattern,
+        '--sort=-committerdate',
+        '--format=%(refname:lstrip=2) %(objectname) %(committerdate:raw)',
+        '--count=%s' % FOR_EACH_REF_MAX_COUNT
+    ]
+    return git("for-each-ref", *command)
+
+
+def get_newer_tagged_revision(tagged_revisions, min_age):
+    # type: (Dict, int) -> Dict
+    epoch = max(i for i in tagged_revisions.keys() if i < min_age)
+    c = tagged_revisions.keys()
+    c.sort(reverse=True)
+    return tagged_revisions[epoch]
+
+
+def list_tagged_revisons(**kwargs):
+    # type: (**Any) -> List[Text]
+    logger.debug("list_tagged_revisons: %s" % kwargs)
+    epoch = COMMAND_ARGS["epoch"]["parser"](**kwargs)
+    # For epoch size equal to weekly we need to use a offset to start
+    # the week on Monday 00:00:00.
+    epoch_offset = 0
+    if epoch == WEEKLY_EPOCH_SIZE:
+        epoch_offset = WEEKLY_EPOCH_OFFSET
+    epoch_threshold = DEFAULT_EPOCH_THESHOLD
+    epoch_until = DEFAULT_EPOCH_UNTIL - epoch_threshold
+    iterator = 0
+    max_count = COMMAND_ARGS["max_count"]["parser"](**kwargs)
+
+    # Iterates the tagged revisions in descending order finding the more
+    # recent commit still older than a "min_age" value.
+    # When a commit is found "min_age" is set to a new value multiplier of
+    # "epoch" but still below of the date of the current commit found.
+    # This needed to deal with intervals where no candidates were found
+    # for the current "epoch" and the next candidate found is yet below
+    # the lower values of the interval (it is the case of J and I for the
+    # interval between Wed and Tue, in the example). The algorithm fix
+    # the next "min_age" value based on the date value of the current one
+    # skipping the intermediate values.
+    # The loop ends once we reached the required number of revisions to return
+    # or the are no more tagged revisions or the min_age reach zero.
+    #
+    #   Fri   Sat   Sun   Mon   Tue   Wed   Thu   Fri   Sat
+    #    |     |     |     |     |     |     |     |     |
+    # -A---B-C---DEF---G---H--IJ----------K-----L-M----N--O--
+    #                                                       ^
+    #                                                      now
+    # Expected result: N,M,K,J,H,G,F,C,A
+
+    def calculate_min_age(until, epoch, offset):
+        return ((((until - offset) / epoch)) * epoch) + offset
```

Comment:
```suggestion
        return ((((until - offset) // epoch)) * epoch) + offset
```

This is needed to work the same in Python 2 and 3.